README.rst

```diff
@@ -278,6 +278,25 @@ differ)::
 
     PASSED (skips=15, successes=1322)
 
+We recommend using the demo which starts 3 federated instances running on ports `8080` - `8082`
+
+    ./demo/start.sh
+
+(to stop, you can use `./demo/stop.sh`)
+
+If you just want to start a single instance of the app and run it directly:
+
+    python -m synapse.app.homeserver \
+      --server-name my.domain.name \
+      --config-path homeserver.yaml \
+      --generate-config \
+      --report-stats=[yes|no]
+
+    synctl start homeserver.yaml
```

Comment:
If you're running it directly via Python why do you need to use `synctl`?

Reply:
AFAICT the first call just creates the config and then exits.

```sh
$ python -m synapse.app.homeserver \
      --server-name my.synapse.server \
      --config-path homeserver.yaml \
      --generate-config \
      --report-stats=no
      
Config file 'homeserver.yaml' already exists. Generating any missing config files.
```

Reply:
Oh I missed the `generate-config` flag there. 👍 I usually just run it directly (`python -m synapse.app.homeserver --config-path homeserver.yaml`) but this should work too.

Reply:
Updated to use python command :+1: